app/domain/authentication/authn_jwt/signing_key/create_signing_key_factory.rb

```diff
@@ -0,0 +1,75 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # Factory that returns the interface implementation of FetchSigningKey
+      CreateSigningKeyFactory ||= CommandClass.new(
+        dependencies: {
+          fetch_provider_uri: Authentication::AuthnJwt::SigningKey::FetchProviderUriSigningKey,
+          fetch_jwks_uri: Authentication::AuthnJwt::SigningKey::FetchJwksUriSigningKey,
```

Comment:
please add `_signing_key` to both params so they match the class they represent.

Reply:
actually - they are regular classes and not command classes so they should also have the `_class` suffix. this will show the reader that it should be called with `.new` and that it is not an instance.

Reply:
Done